v4.13/graph.yaml

```
@@ -57,6 +63,10 @@ schema: olm.bundle
 image: registry.redhat.io/rhtas/rhtas-operator-bundle@sha256:d423c8f491a008886926520e12fdd86a5878ce6b5b169100ba495689cff42f2c
 # rhtas-bundle-registry v1.1.0
 ---
+schema: olm.bundle
+image: registry.redhat.io/rhtas/rhtas-operator-bundle-v1-1@sha256:4570fd896ab783db81ba07d8f3bb3e024bc8f9ee9364a3dba8e2a5d47f879398
```

```suggestion
image: registry.redhat.io/rhtas/rhtas-operator-bundle@sha256:4570fd896ab783db81ba07d8f3bb3e024bc8f9ee9364a3dba8e2a5d47f879398
```

we dont need the `-v1-1` added on here :)

---

ok sry I am a bit confused where to set the version name behind the bundle and where not :D Thanks for your help! I changed it :)